x-pack/plugins/entity_manager/server/lib/entities/built_in/services_from_ecs_data.ts

```diff
@@ -10,14 +10,14 @@ import { BUILT_IN_ID_PREFIX } from './constants';
 
 export const builtInServicesFromEcsEntityDefinition: EntityDefinition =
   entityDefinitionSchema.parse({
-    version: '0.4.0',
+    version: '0.5.0',
     id: `${BUILT_IN_ID_PREFIX}services_from_ecs_data`,
     name: 'Services from ECS data',
     description:
       'This definition extracts service entities from common data streams by looking for the ECS field service.name',
     type: 'service',
     managed: true,
-    indexPatterns: ['logs-*', 'filebeat*', 'traces-apm*'],
+    indexPatterns: ['logs-*', 'filebeat*', 'traces-*'],
```

Comment:
Q: Should we add metrics as well? I see here we are checking for [metrics](https://github.com/elastic/kibana/blob/61251bfdaffdb621558fff96d78cc8b2260c0abe/x-pack/plugins/observability_solution/apm/public/utils/get_signal_type.ts) as well
cc @cauemarcondes

Reply:
Yes, I tested it, I just added a screenshot with the otel service showing in the inventory

<img width="1913" alt="image" src="https://github.com/user-attachments/assets/758e9992-2dbf-443f-b62f-99ef1154dc6f">

Reply:
Users could have only metrics, and in that case, they would **not** see any service on the Inventory. I feel like this is an edge case though.

Reply:
@cauemarcondes That's fair, thanks for explaining. 
@MiriamAparicio Thank you, I will also test with the service overview and add a comment :)

Reply:
The service overviews are fine as well 💯  (the logs charts are also there for the PoC service but I think that's expected): 
<img width="1366" alt="image" src="https://github.com/user-attachments/assets/8957483c-c67f-4b73-9174-8bc99224cf82">
![image](https://github.com/user-attachments/assets/751917a4-9077-48ee-b136-075c2fd8b920)